Headlines/src/Controllers/FilterViewController.swift

```diff
@@ -0,0 +1,156 @@
+//
+//  FilterViewController.swift
+//  Headlines
+//
+//  Created by Ezequiel Becerra on 9/8/17.
+//  Copyright © 2017 Ezequiel Becerra. All rights reserved.
+//
+
+import UIKit
+
+class FilterViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegate, UIGestureRecognizerDelegate {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/e8a9d597da6040db016ce3925847693be5c6764f/Headlines/src/Controllers/FilterViewController.swift#L11"></span>Line should be 120 characters or less: currently 129 characters</td>
    </tr>
  </tbody>
</table>